src/com/sun/ts/tests/jaxrs/platform/beanvalidation/annotation/build.xml

```diff
@@ -18,19 +18,66 @@
 
 <project name="jaxrs_platform_beanvalidation_annotation" basedir="." default="usage">
 
-	<property name="app.name" value="jaxrs_platform_beanvalidation_annotation" />
+    <import file="../../../common/common.xml" />
```

Comment:
@brideck The reordering of elements makes it harder to read the pull request.  Also, changes to whitespace are also harder to review.

Reply:
Understood.  Just trying to take the opportunity to give the file consistent formatting and move imports to the top.  These files were not easy to look at when rendered in my editor as they were.